00_Base/src/interfaces/centralsystem/NetworkConnection.ts

```diff
@@ -6,6 +6,7 @@ export interface INetworkConnection {
     addOnConnectionCallback(onConnectionCallback: (identifier: string, info?: Map<string, string>) => Promise<boolean>): void;
     addOnCloseCallback(onConnectionCallback: (identifier: string, info?: Map<string, string>) => Promise<boolean>): void;
     addOnMessageCallback(onMessageCallback: (identifier: string, message: string, info?: Map<string, string>) => Promise<boolean>): void;
+    addSentMessageCallback(onSentMessageCallback: (identifier: string, message: string, error?: any, info?: Map<string, string>) => Promise<boolean>): void;
```

Comment:
Receive a compile error on `error?: any` when running `./unix-init-install-all.sh` but docker build command finishes without any issues. How about change to `object` or `object | string` ? 
```
/Users/zcheng/Documents/DEV/citrineos-core/00_Base/src/interfaces/centralsystem/NetworkConnection.ts
  9:97  error  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
```